bin/main.ml

```diff
@@ -114,28 +115,34 @@ end
 
 type target =
   | File      of Path.t
+  | Alias     of Path.t
   | Alias_rec of Path.t
 
+let parse_alias path ~contexts =
+  let dir = Path.parent_exn path in
+  let name = Path.basename path in
+  match Path.extract_build_context dir with
+  | None -> (contexts, dir, name)
+  | Some ("install", _) ->
+    die "Invalid alias: %s.\n\
```

Comment:
Would it be useful to pass a `loc` to have a nicer error message? (or return a `result` from here)

Reply:
This is from the command line, so we don't have a loc for it.